scene/resources/particle_process_material.cpp

```diff
@@ -882,73 +883,63 @@ void ParticleProcessMaterial::_update_shader() {
 			code += "		}\n";
 		}
 	}
-	code += "		}\n";
-	code += "	process_display_param(params, 0.);\n";
-	code += "//	process_dynamic_parameters(dynamic_params, 0., alt_seed, TRANSFORM, EMISSION_TRANSFORM, DELTA);\n";
+	code += "	}\n\n";
+	code += "	process_display_param(params, 0.0);\n\n";
 	code += "	VELOCITY = (EMISSION_TRANSFORM * vec4(VELOCITY, 0.0)).xyz;\n";
 	code += "	VELOCITY += EMITTER_VELOCITY * inherit_emitter_velocity_ratio;\n";
 	if (particle_flags[PARTICLE_FLAG_DISABLE_Z]) {
-		code += "		VELOCITY.z = 0.;\n";
-		code += "		TRANSFORM[3].z = 0.;\n";
+		code += "	VELOCITY.z = 0.0;\n";
+		code += "	TRANSFORM[3].z = 0.0;\n";
 	}
-	code += "}\n";
-	code += "\n";
+	code += "}\n\n";
 
 	code += "void process() {\n";
 	code += "	uint base_number = NUMBER;\n";
-	// TODO add optional determinism here
-	code += "//	if (repeatable){\n";
-	code += "//		base_number = INDEX;\n";
-	code += "//	}\n";
+	// TODO: Add optional determinism here.
+	code += "	//if (repeatable) {\n";
+	code += "	//	base_number = INDEX;\n";
+	code += "	//}\n";
```

Comment:
Should we remove the dead code and only keep the TODO in cpp? Or even remove the TODO?